tasks/ast_tools/src/generators/visit.rs

```diff
@@ -525,6 +527,18 @@ impl<'a> VisitBuilder<'a> {
                     };
                     enter_scope_at = ix;
                 }
+                if have_exit_scope {
+                    assert!(
+                        exit_scope_at.is_none(),
+                        "Scopes cannot be exited more than once. Remove the extra `#[scope(exit_before)]` attribute(s)."
+                    );
+                    let scope_leave = &scope_events.1;
+                    result = quote! {
+                        #scope_leave
+                        #result
+                    };
```

Comment:
Rename var for consistency.

```suggestion
                    let scope_exit = &scope_events.1;
                    result = quote! {
                        #scope_exit
                        #result
                    };
```